rest/config.go

```diff
@@ -1040,6 +1041,12 @@ func RunServer(config *ServerConfig) {
 
 	go sc.PostStartup()
 
+	// If cookie override == true OR cookie override isn't set and using tls serving over tls
+	// Otherwise usingTLS will be set to false by default
+	if (config.TLSSecureCookieOverride != nil && *config.TLSSecureCookieOverride) || (config.TLSSecureCookieOverride == nil && config.SSLCert != nil) {
```

Comment:
Rather than doing this here and storing in the ServerContext, we could add a method on the ServerConfig struct that will check if the override is non-nil (and return the value of that), otherwise return the value of `config.SSLCert != nil` - which we can call directly from `makeSessionWithTTL`

Reply:
I guess your method of calculating that might be a bit easier to read 😛